src/domain/getCertificate.ts

```diff
@@ -4,7 +4,7 @@ import CertificateDetails from '../interfaces/CertificateDetails';
 import getObjectFromS3 from '../infrastructure/s3/s3Service';
 import validate from '../utils/validationService';
 import NoBodyError from '../errors/NoBodyError';
-import CertificateNumberError from '../errors/CertificateNumberError';
+import TestNumberError from '../errors/TestNumberError';
```

Comment:
Something to discuss  as devs but I am thinking that we could be using barrel files with a named import where it is consumed so we avoid all these individual imports. I suspect that the list could grow. What do you think please?

Reply:
By "barrel files" do you mean having multiple error classes in the same file and then named exports? Cause I'm fine with that.

Reply:
was that a request for me to do that now, or is it alright as is?